### PR TITLE
Add be.telenet.user.QSSTV

### PR DESCRIPTION
**Pull request by AsciiWolf**
This is a continuation of work originally done by @lu9dce [here](https://github.com/flathub/flathub/pull/1973). The Flatpak works fine and should be ready for merge.

@barthalion already asked me if I want to take over this Flatpak. I originally refused, but I changed my mind. :-)

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build be.telenet.user.QSSTV

File: be.telenet.user.QSSTV.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+       - make -j4
+       - make install DESTDIR=$FLATPAK_DEST
+    sources:
+      - type: archive
+        url: https://github.com/Hamlib/Hamlib/releases/download/3.3/hamlib-3.3.tar.gz
+        sha256: c90b53949c767f049733b442cd6e0a48648b55d99d4df5ef3f852d985f45e880
+  - name: fftw
+    buildsystem: simple
+    build-commands:
+       - ./configure --prefix=/ --libdir=/lib --enable-shared --enable-static --enable-threads
```
Comment (hfiguiere) [inline be.telenet.user.QSSTV.yml:38]: Prefix `/` is just wrong IMHO. Using `DESTDIR=$FLATPAK_DEST` is probably not what you want. `$FLATPAK_DEST` is the prefix and DESTDIR is something used for staging.

File: be.telenet.user.QSSTV.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+  - /bin/usr
+modules:
+  - name: hamlib
+    buildsystem: simple
```
Comment (hfiguiere) [inline be.telenet.user.QSSTV.yml:26]: what's wrong with using `buildsystem: autotools` here?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by AsciiWolf** [comment]
@xylo04 Let us know if you are interested in co-maintaining this Flatpak.

**Comment by AsciiWolf** [comment]
bot, build be.telenet.user.QSSTV

**Comment by AsciiWolf** [comment]
bot, build be.telenet.user.QSSTV

File: be.telenet.user.QSSTV.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      - type: archive
+        url: http://www.fftw.org/fftw-3.3.9.tar.gz
+        sha256: bf2c7ce40b04ae811af714deb512510cc2c17b9ab9d6ddcf49fe4487eea7af3d
+
+  - name: qsstv
+    buildsystem: simple
+    build-commands:
+      - mkdir -p $FLATPAK_DEST/share/applications/
+      - mkdir -p $FLATPAK_DEST/share/icons/hicolor/128x128/apps/
+      - cp $FLATPAK_BUILDER_BUILDDIR/qsstv/icons/qsstv.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/be.telenet.user.QSSTV.png
```
Comment (AsciiWolf) [inline be.telenet.user.QSSTV.yml:57]: The icon directory is probably incorrect. The actual icon size seems to be 42x42. I will fix this.

File: be.telenet.user.QSSTV.yml (hunk 2 above)
Comment (AsciiWolf) [reply be.telenet.user.QSSTV.yml]: Thanks. I have changed it to `buildsystem: cmake-ninja`.

File: be.telenet.user.QSSTV.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      - qmake QMAKE_LFLAGS="-L/app/lib" QTC_PREFIX="/app"
+      - make -j4
```
Comment (barthalion) [inline be.telenet.user.QSSTV.yml:59]:
```suggestion
      - make -j$FLATPAK_BUILDER_N_JOBS
```

Comment (AsciiWolf) [reply be.telenet.user.QSSTV.yml]: Thanks, I overlooked this. :)

File: be.telenet.user.QSSTV.yml (hunk 3 above)
Comment (barthalion) [reply be.telenet.user.QSSTV.yml:57]: Just this one left and it's ready to go, unless you can find a bigger icon in general.

Comment (AsciiWolf) [reply be.telenet.user.QSSTV.yml:57]: Is `$FLATPAK_DEST/share/icons/hicolor/42x42/apps/` a valid location supported by Flatpak? I am not sure.

Comment (barthalion) [reply be.telenet.user.QSSTV.yml:57]: 48x48 if anything, I doubt 42x42 would work.

**Comment by AsciiWolf** [comment]
bot, build be.telenet.user.QSSTV

**Comment by xylo04** [comment]
Sure, I'll be a co-maintainer.

**Pull request closed without merging**